assets/agenda/components/AgendaPreview.tsx

```diff
@@ -90,7 +91,10 @@ class AgendaPreview extends React.PureComponent<any, any> {
                         )}
                     >
                         <hgroup>
-                            <TopStoryLabel item={item} config={previewConfig} size='big' />
+                            <div className='d-flex gap-2'>
```

Comment:
might be useful to have some `LabelGroup` component for this

Reply:
maybe even some `AgendaLabels` component otherwise it seems like duplicated code all over

Reply:
so do i need to add some LableGroup component in this pr?

Reply:
@petrjasek what about this one?